### DISCUSSION TITLE
Not Able to connect to one of my server via ssh after latest update.

### DISCUSSION THREAD

**Discussion by rishabkshk** [codefori/vscode-ibmi#1333] [General]
Getting the below error while trying to connect. Is there anything wrong with my server?

![image](https://github.com/halcyon-tech/vscode-ibmi/assets/21074746/83a9c175-af17-4a87-9219-48bedef6ac34)

**Comment by worksofliam** [comment]
Hi @rishabkshk 

Check out [Unexpected packet before version](https://halcyon-tech.github.io/docs/#/pages/tips/setup?id=unexpected-packet-before-version).

**Reply by sebjulliand** [reply]
You beat me by one second 🤣

**Reply by rishabkshk** [reply]
Thanks!

**Comment by sebjulliand** [comment]
Hello,
indeed, there seems to be a script in your home directory missing with your connection.

Please check this documentation page about how to handle this: https://halcyon-tech.github.io/docs/#/pages/tips/setup?id=unexpected-packet-before-version

**Reply by rishabkshk** [reply]
Thanks!

**Reply by rishabkshk** [reply]
Hi Sebastian,

I tried to find the .bashrc and .profile files but not able to. Could you please possibly point me to the right path/folder?

**Reply by worksofliam** [reply]
@rishabkshk Are you using 1.9.2 also?

**Reply by rishabkshk** [reply]
@worksofliam Yes!

**Reply by worksofliam** [reply]
@rishabkshk And you are still getting the same issue?

You can find `.bashrc` or `.profile` in your users home directory.

**Reply by rishabkshk** [reply]
Well for my user, there is no home directory. Should I create one?

**Reply by rishabkshk** [reply]
> Well for my user, there is no home directory. Should I create one?

But also the wrkusrprf shows /home/MYUSER but it does not exist on IFS.

**Reply by worksofliam** [reply]
Sure, go ahead and create it.

**Reply by rishabkshk** [reply]
It worked! I am able to login but this didn't happen earlier before the update. Any idea what could have caused it?
